src/api/getAllData.tsx

```diff
@@ -0,0 +1,272 @@
+import { getTeamsForOrg } from "./getTeamsForOrg";
+import { getReposForTeam } from "./getReposForTeam";
+import { getVulnsFromRepo } from "./getVulnsFromRepo";
+
+import { Org, Team, Topic, Repository} from "../utils/types";
+import { formatVulnData } from "../utils/functions";
+
+export const getAllData = (graphql:any,  orgLogin: string) => {
+  return getAllRawData(graphql, orgLogin)
+};
+
+export async function getAllRawData(graphql:any, orgLogin:string): Promise<any> {
```

Comment:
Try not to go back and forth between the arrow function operator and `function`

Reply:
This function is pretty hefty. I'd suggest breaking it up into smaller helper functions

Reply:
This function return `Org`. You should have the type here as `Promise<Org>`